code/modules/mob/living/simple_animal/hostile/headcrab.dm

```diff
@@ -191,3 +198,363 @@
+	var/gonarch_rampage_trigger = 30
+	/// The current mode its statemachine is in. Currently it has: Rest, Defending, Returning and Newhome (Looking for a new home) as well as Rampage.
+	var/mode = GONARCH_MODE_REST
+	/// The Gonarch Children list is be used to see how many headcrabs it has born. - It will be created with LazyAdd later.
+	var/list/gonarch_children = null
+	/// The Maximum amount of Children that the Gonarch will create. It also checks if its babies died so ultimately will always produce headcrabs up to this number.
+	var/desired_child_count = 10
+	/// The world.time that it last succesfully created a headcrab on as well as the spawn frequency. These are currently set to 1 every minute.
+	var/time_last_babies = 0
+	/// The Spawn frequency of the headcrabs. Currently its set to 1 every minute.
+	var/gonarch_headcrab_spawn_frequency = 600
+	/// The Frustration goes up, if the Gonarch makes no progress getting to its home. If it reaches the limit designated it will start to destroy more, but also cause more homesick.
+	var/frustration = 0
+	/// The limit of frustration it accepts before starting to destroy more stuff around it.
+	var/frustration_limit = 3
+	/// The range in which it looks for a suitable nest.
+	var/gonarch_nestfind_range = 13
+	/// The Screeching Range defines until what range carbons are effected by its stunning screen.
+	var/gonarch_screech_range = 8
+	/// This tracks how many times it looked for a new home, and sets a limit to its attempts. If this limit is reached, even rooms with light are now okay for the gonarch to use.
+	var/gonarch_finding_home_attempts = 0
+	/// The amount of attempts it is willing to look for a new home. If it reaches this number, the Gonarch will also consider areas with lights as its new breeding ground.
+	var/gonarch_finding_home_limit_attempts = 2
+	/// The cooldown time for the ranged attack.
+	var/gonarch_rangeattack_cooldown = 200
+	/// The amount of tiles the artillery fires around its target
+	var/gonarch_rangeattack_range = 4
+	/// Last attack in world.time that the gonarch used its range attack.
+	var/gonarch_rangeattack_lastattack = 0
+	/// The gonarch selects 24 turfs around its target and its acid attack randomly selects those turfs to have the acid attack land on. Setting this to 100 guarantees that all turfs aroudn the target will receive acid.
+	var/gonarch_rangeattack_turfchance = 33
+	/// The interval in deciseconds that the gonarch will do a round acid attack while returning.
+	var/gonarch_rangeattack_swoop_interval = 5
+	/// Last worldtime that the gonarch used a rangeswoop while returning
+	var/gonarch_rangeattack_swoop_lastattack = 0
+	/// Range of the walking home ranged attack
+	var/gonarch_rangeattack_swoop_range = 9
+	/// These are the soundlists of the gonarch. They are created null and will be filled when the gonarch is spawned to save memory and the hidden init proc.
+	var/list/gonarch_soundlist_rampage = null
```

Comment:
Can remove all the `= null` parts since they are initialized in the `Initialize` proc and having them set to `null` here won't add any value/clarity.
Can also shorten the comment that way